packages/appstore/src/components/modals/wallets-migration-failed/wallets-migration-failed.tsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import { Text, useOnClickOutside, Modal, Button } from '@deriv/components';
+import { localize } from '@deriv/translations';
+import { useStore, observer } from '@deriv/stores';
+import './wallets-migration-failed.scss';
+
+const WalletsMigrationFailed = observer(() => {
+    const { traders_hub, ui } = useStore();
+    const { is_wallet_migration_failed, setWalletsMigrationFailedPopup } = traders_hub;
+    const { is_mobile } = ui;
+
+    const mobile_add_class = is_mobile ? '--mobile' : '';
+
+    const wallets_migration_failed_ref = React.useRef<HTMLDivElement>(null);
+
+    const handleClose = () => {
+        setWalletsMigrationFailedPopup(false);
+    };
+
+    const handLiveChatButtonClick = () => {
+        window.LC_API?.open_chat_window();
+        setWalletsMigrationFailedPopup(false);
+    };
+
+    const validateClickOutside = (e: MouseEvent) => {
+        return is_wallet_migration_failed && !wallets_migration_failed_ref?.current?.contains(e.target as Node);
+    };
+
+    useOnClickOutside(wallets_migration_failed_ref, handleClose, validateClickOutside);
+
+    return (
+        <React.Fragment>
+            {is_wallet_migration_failed && (
+                <Modal
```

Comment:
@sergei-deriv Not really sure but I think you won't need to conditionally render the `Modal` component, If `is_open` is false it won't be added to the dom I guess, So it's handled by itself. Better to check with someone who knows better tho 😅 You can ignore this comment 🙇🏻

Reply:
@farzin-deriv Awesome! Kindly ask always review my PRs :)